tests/run-make-fulldeps/hotplug_codegen_backend/Makefile

```diff
@@ -6,8 +6,11 @@ include ../../run-make/tools.mk
 # backends and that this external codegen backend is only included in the dep info if
 # -Zbinary-dep-depinfo is used.
 
+REQUIRED := echo # This test requires `echo` to exist
```

Comment:
Can you help me understand what echo is used for in the test? It seems like the previous implementation presumably wanted specifically /bin/echo, not some other path... But I'm not seeing where that usage happens.

Asking the user to install echo seems basically useless, I don't think anyone who doesn't have it is likely to be willing/able to install it...

Reply:
Ah, I see the discussion thread now. I would recommend we drop this check and try and merge. Maybe that finds something, otherwise we can, well, just not have the check.

Reply:
Sure, I've removed the check.